### DISCUSSION TITLE
GitHub Education upgrade approved and "upgraded" but not actually applied

### DISCUSSION THREAD

**Discussion by hi-liang** [community/community#75585] [GitHub Education]
### Hi everyone,

Hi, this might be similar to #66773 and/or #68971. 

Our previous Teams organization was downgraded to Free some weeks ago, possibly due to lack of updated verification. I submitted a request on Oct 25th and got the popup approval immediately 
![image](https://github.com/community/community/assets/3301067/6a33bd66-9f80-4442-b2a2-4bc651e7edfc)

But this still doesn't seem to have taken effect. Another member of our org also submitted a request a few days later and received an email on Nov 7:

![image](https://github.com/community/community/assets/3301067/57ea5ad1-7b7d-44cf-b5c1-c498eca52d1a)

But once again, our org is still on Free tier.  Strangely at the same time as that email on our Audit Logs, we have an event:

```
picoCTF – account.plan_change
  Switched the picoCTF organization from the free_with_addons to the free plan
```

Which unfortunately doesn't show the direction we hoped for, from "free" to "teams" :(

Thanks for your time

**Comment by goyalvipul** [comment]
Hi.  I am also facing the same issue.

**Comment by ProfessorAkram** [comment]
same
